openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -27,27 +27,20 @@ class CreateSaver(NewCreator):
     description = "Fusion Saver to generate image sequence"
     icon = "fa5.eye"
 
-    instance_attributes = [
-        "reviewable"
-    ]
+    instance_attributes = ["reviewable"]
```

Comment:
Haha, I love this. I love his people disagree on this styling - last commit that adjusted this made it the opposite [here](https://github.com/ynput/OpenPype/commit/d637fed33de88d4048c40b56ecf3f5cb86aab654).

So 3, 2, 1...

Let the battle begin.

@Sponge96 versus @jakubjezek001 🥊

Reply:
Black did this haha, I really don't have a strong opinion here

Reply:
I'll mark as resolved as I don't have a strong opinion here. Feel free to request a change back (or change it back) if any of you feel strongly that Black is incorrect haha